tools/wasm-sourcemap.py

```diff
@@ -20,6 +20,10 @@
 from subprocess import Popen, PIPE
 import sys
 
+sys.path.insert(1, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Is there a reason for the `1` here rather than `0`?  I've seen both in the codebase but I would expect `0` to be what you want normally.

Reply:
Oh, I don't know - I just copy-pasted that.

The order determines precedence I assume - but should be fine either way here. In fact we could just append I guess?